src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -25,7 +25,11 @@ public class ParameterProvider {
   public static final String MAX_MEMORY_LIMIT_IN_BYTES = "MAX_MEMORY_LIMIT_IN_BYTES".toLowerCase();
   public static final String ENABLE_PARQUET_INTERNAL_BUFFERING =
       "ENABLE_PARQUET_INTERNAL_BUFFERING".toLowerCase();
+  // This is actually channel size limit at this moment until we implement the size tracking logic
+  // at table/chunk level
   public static final String MAX_CHUNK_SIZE_IN_BYTES = "MAX_CHUNK_SIZE_IN_BYTES".toLowerCase();
```

Comment:
does it make sense to rename this to `MAX_CHANNEL_BUFFER_SIZE_IN_BYTES`?

Reply:
Makes sense, let me update